multiplex_test.go

```diff
@@ -394,6 +394,38 @@ func TestClosing(t *testing.T) {
 	}
 }
 
+func TestCloseChan(t *testing.T) {
+	a, b := net.Pipe()
+
+	mpa := NewMultiplex(a, false)
+	mpb := NewMultiplex(b, true)
+
+	ctx, cancel := context.WithTimeout(context.Background(), time.Second*5)
+	defer cancel()
+
+	_, err := mpb.NewStream(ctx)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	_, err = mpa.Accept()
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	go func() {
+		time.Sleep(time.Second * 1)
```

Comment:
Why do we need a `time.Sleep` here? This could be executed immediately, right?

Reply:
Yes, we don't need the `.Sleep` here.

Reply:
Fixed in force push

Reply:
huh, GitHub doesn't work like bitbucket does. It doesn't seem to update from force push.

Reply:
my apology, I clicked to use the wrong branch as the source. I will close this one and use the correct branch.